.bazelci/presubmit.yml

```diff
@@ -14,7 +14,39 @@ tasks:
       # These tests are currently incompatible with RBE
       - "-//tests/integration:UnsafeSharedCacheTest"
       - "-//tests/integration/override_targets"
-  ubuntu1804:
+  ubuntu1804_0_25_3:
+    platform: ubuntu1804
+    bazel: 0.25.3 
+    shell_commands:
+      - bazel run @unpinned_regression_testing//:pin
```

Comment:
I'm unfamiliar with including both shell_commands and test_targets. What happens? Does shell_commands get run as part of setup?

Reply:
Yeah: https://buildkite.com/bazel/rules-jvm-external/builds/667#ac7a4379-d6a8-4a2b-b977-068aca09691d/140